docs/도메인_유스케이스.md

```
@@ -0,0 +1,162 @@
+
+단독 수정을 무조건 금지합니다
+
+## 도메인
+
+---
+
+### 가게 관리
+
+- **점주는 가게를 생성할 수 있다**
+    - 가게 생성은 가게 이름, 위치, 상세 설명, 대표 이미지, 대학 이름, 테이블 점유비(시간 단위,비용)의 정보로 생성할 수 있다.
+    - 가게 이름, 위치, 대표 이미지, 대학 이름, 테이블 비용은 필수 값이다. → 위치 중복 여부 검증?
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)